content/applications/sales/sales/products_prices/products/product_images.rst

```diff
@@ -1,130 +1,168 @@
-===================================================
-Automatically get product images with Google Images
-===================================================
+=================================
+Product images with Google Images
+=================================
 
-The product images are very useful in Odoo, for example, to quickly find a product or check if you
-scanned the right one, but it can be a bit painful to set up especially if you have a lot of
-products. **Google Custom Search** allows finding images automatically for your product, based on
-their barcode, keeping your focus on what matters in your business.
+Having appropriate product images in Odoo is useful for a number of reasons. However, if a lot of
+products need images, it can become incredibly time-consuming.
+
+But, with *Google Custom Search* API configured within the Odoo database, finding product images for
+products (based on their barcode) is extremely efficient.
 
 .. _product_images/configuration:
 
 Configuration
 =============
 
-This functionnality requires configuration both on Google and on Odoo.
+In order to utilize *Google Custom Search* within an Odoo database, configuration **must** be
+completed on both -- Google and Odoo.
 
-With a free Google account, you can get up to 100 free images per day. If you need a higher rate,
-you'll have to upgrade to a billing account.
+.. note::
+   Free Google accounts allow up to 100 free images per day. If a higher amount is needed, a billing
+   upgrade is required.
 
 .. _product_images/google-api-dashboard:
 
 Google API dashboard
 --------------------
 
 #. Go to the `Google Cloud Platform API & Services <https://console.developers.google.com/>`_ page
-   to generate Google Custom Search API credentials. Log in with your Google account.
+   to generate Google Custom Search API credentials. Then, log in with a Google account. Next, agree
+   to their :guilabel:`Terms of Service` by checking the box, and clicking :guilabel:`Agree and
+   Continue`.
 
-#. Select or create an API project to store the credentials. Give it an explicit name
-   (e.g. Odoo Images).
+#. Select (or create) an API project to store the credentials. Give it a memorable
+   :guilabel:`Project Name`, select a :guilabel:`Location` (if any), then click :guilabel:`Create`.
 
-#. In the credentials section, click on **Create Credentials** and select **API Keys**.
+#. With the :guilabel:`Credentials` selected in the left sidebar. Click :guilabel:`Create
+   Credentials`, and selecting :guilabel:`API key` from the drop-down menu.
```

Comment:
This section doesn't make sense to me. I think it needs to be rewritten. What was there originally makes more sense.

Reply:
I cleaned it up a bit. Thanks for pointing this out.